tests/cargo-add.rs

```diff
@@ -820,3 +819,20 @@ fn add_prints_message_for_build_deps() {
         .prints_exactly("Adding hello-world v0.1.0 to build-dependencies")
         .unwrap();
 }
+
+#[test]
+fn add_typo() {
```

Comment:
Does this test require an internet connection? Can it fail spuriously on travis in that case?

Reply:
Ah. I've just discovered the `test-external-apis` feature. We already use this to guard other UTs that make network requests. This seems like the correct thing - it's turned on in travis, but developers need to opt-in when running `cargo test`